vagrant/README.md

```diff
@@ -66,7 +66,7 @@ tasks that are dependant on the existance of the variable 'my_cool_var'.
    The default port is 35675.
 1. Run `vagrant up`
 
-## Testing QPID
+## Testing QPID [Deprecated since Candlepin v4.0.0]
```

Comment:
Same here: no point keeping this. Just remove the section entirely

Reply:
Removed.